### ISSUE TITLE
Temp folder issue (Mac Os X).

### ISSUE THREAD

**Issue by Vincz** [debashish2014/local-package-publisher#3]
Hi guys.
Following the documentation, I tried to globally publish a package name **super** running `local-package-publisher --publish` in the project root directory.
I'm on Mac Os X (with node version v15.14.0) and get an error like the following:

````
super package published successfully to global
To consume this package, run npm link super in target project
node:internal/bootstrap/switches/does_own_process_state:112
  rawMethods.chdir(directory);
             ^

Error: ENOENT: no such file or directory, chdir '/Volumes/x/dev/super' -> '/var/folders/vf/9bptzv4d4cxfym7g6bt9yyrw0000gn/T/tmp-60750SDC706yNP5V'
    at process.wrappedChdir (node:internal/bootstrap/switches/does_own_process_state:112:14)
    at process.chdir (/Users/me/.nvm/versions/node/v15.14.0/lib/node_modules/local-package-publisher/node_modules/graceful-fs/polyfills.js:22:11)
    at /Users/me/.nvm/versions/node/v15.14.0/lib/node_modules/local-package-publisher/index.js:138:17
    at new Promise (<anonymous>)
    at linkDirGlobal (/Users/me/.nvm/versions/node/v15.14.0/lib/node_modules/local-package-publisher/index.js:129:12)
    at /Users/me/.nvm/versions/node/v15.14.0/lib/node_modules/local-package-publisher/index.js:324:13 {
  errno: -2,
  code: 'ENOENT',
  syscall: 'chdir',
  path: '/Volumes/x/dev/super',
  dest: '/var/folders/vf/9bptzv4d4cxfym7g6bt9yyrw0000gn/T/tmp-60750SDC706yNP5V'
}
````

The `.local-pack` folder is created and contains the `settings.json` file with the following content:
```json
{"TempPath":"/var/folders/vf/9bptzv4d4cxfym7g6bt9yyrw0000gn/T/tmp-60750SDC706yNP5V"}
```

It's like there is a problem with the temp folder, it doesn't seem to be created. 

Any idea? Thanks!

**Comment by robertocerdasilva** [comment]
Same problem in WIndows 10....

**Comment by gfina** [comment]
problem also exists in node 16.15

**Comment by batyaro** [comment]
Same problem to me in Windows. any answer?

**Comment by PatrykMilewski** [comment]
same problem here

**Comment by Rogatto** [comment]
Anyone has any idea how to fix this problem? @debashish2014

**Comment by debashish2014** [comment]
I'm sorry for the delay in fixing the issue. I was not getting time to maintain this project because of other projects' workloads. This issue has been resolved in version 1.0.5. Can you please try this version and share your thoughts?

Closing the issue for now.
